Dockerfile

```diff
@@ -0,0 +1,55 @@
+# https://docs.docker.com/samples/rails/
+FROM ruby:2.7.3
+ 
+WORKDIR  /jinglejam
+
+COPY Gemfile /jinglejam/Gemfile
+COPY Gemfile.lock /jinglejam/Gemfile.lock
```

Comment:
these could be done in one line as 

COPY Gemfile Gemfile.lock /jinglejam/

this way we will only be creating one layer in the docker image

Reply:
This has been done!